rasa/shared/core/events.py

```diff
@@ -458,6 +458,10 @@ def __init__(
         }
 
         if parse_data:
+            if "text_tokens" in parse_data:
```

Comment:
Do we have a constant for the `"text_tokens"` key?

Reply:
Yup, it's [here](https://github.com/RasaHQ/rasa/blob/819c339f3f5cdf32593cc725ff7d8f4a8253ea3a/rasa/nlu/constants.py#L55) - would replacing this with `TOKENS_NAMES.get(rasa.shared.nlu.constants.TEXT)` be alright?